### DISCUSSION TITLE
digi-locker certificate-api mount path issue

### DISCUSSION THREAD

**Discussion by gohilamariappan** [Sunbird-RC/community#1040] [Issues] [answered]
Hi Team!

we are getting an error in certificate api deployment. below are the logs.
while checking this issue, am unable to find the mount path for config file.
https://github.com/Sunbird-RC/sunbird-rc-core/blob/main/docker-compose.yml#L433
@holashchand please suggest
################
Failed to find configuration ./config/application-default.yml
Error : open ./config/docType.json: no such file or directoryError occurred while reading json file :  invalid argumentError Unmarshalling Json file: unexpected end of JSON inputFailed parsing log level[GIN-debug] [WARNING] Running in "debug" mode. Switch to "release" mode in production.
 - using env:   export GIN_MODE=release
 - using code:  gin.SetMode(gin.ReleaseMode)
###########

Regards,
Gohila

**Comment by holashchand** [comment, marked as answer]
Hi, @gohilamariappan , digilocker-certificate-api service is moved [here](https://github.com/Sunbird-RC/auxillary-services/tree/main/digilocker-certificate-api)

the desired files required, can be taken from here
